reel-picker/src/main/java/com/innowisegroup/reelpicker/datetime/LocalTime.kt

```diff
@@ -15,13 +15,35 @@ class LocalTime private constructor(val hour: Int, val minute: Int) : Serializab
 
     override fun hashCode(): Int = 31 * hour + minute
 
+    fun plusHours(hoursToAdd: Int): LocalTime =
```

Comment:
this implementation differs from java.time. Take a look in source code. Test prove it

Reply:
I made like this (taken from sources)

![image](https://user-images.githubusercontent.com/79689735/186515100-1dbcab19-4489-4715-a199-c4aa539a5459.png)